metadata-ingestion/src/datahub/ingestion/source/iceberg/iceberg.py

```diff
@@ -145,6 +146,13 @@ def _get_datasets(self, catalog: Catalog) -> Iterable[Identifier]:
         self.report.report_no_listed_namespaces(len(namespaces))
         tables_count = 0
         for namespace in namespaces:
+            namespace_repr = ".".join(namespace)
+            if not self.config.namespace_pattern.allowed(namespace_repr):
+                LOGGER.info(
```

Comment:
This all cap LOGGER looks scary but I guess it was like this before :)

Reply:
Yeah I just followed what seems to be established convention in `datahub` code-base. I guess somebody used caps first since it was a variable defined outside of a class (on module-level).